### PR TITLE
Removed Deprecated checkLogList Method

### PR DESCRIPTION
**Pull request by nixon124**

### CODE REVIEW

File: src/test/java/emissary/test/core/junit5/LogbackTesterTest.java [hunk 1]
```diff
@@ -90,13 +91,6 @@ void canDetectWrongThrowableMessage() {
...
         assertNotNull(logBackTester);
-        boolean[] throwing = new boolean[1];
-        String[] messages = new String[1];
-        Level[] levels = new Level[1];
-        levels[0] = Level.WARN;
-        messages[0] = "Not good news.";
-        throwing[0] = false;
-        logger.warn("Not good news.");
-        logBackTester.checkLogList(levels, messages, throwing);
+        logBackTester.checkLogList(Collections.emptyList());
```
Comment (drivenflywheel) [inline src/test/java/emissary/test/core/junit5/LogbackTesterTest.java:94]: This isn't an equivalent change. It should be checking a list with one element, not an empty list. (The `logger.warn("Not good news")` line should not have been deleted.)

File: src/test/java/emissary/test/core/junit5/LogbackTesterTest.java [hunk 2]
```diff
@@ -90,13 +91,6 @@ void canDetectWrongThrowableMessage() {
     void canCheckOneEvent() {
```
[the remark this answers is not in the document]
Comment (drivenflywheel) [reply src/test/java/emissary/test/core/junit5/LogbackTesterTest.java:91]: No, it should be fixed.  Shouldn't have been gutted from 1 logged messages to 0

[the remark this answers is not in the document]
Comment (nixon124) [reply src/test/java/emissary/test/core/junit5/LogbackTesterTest.java:91]: I might be wrong, but I think the point of this test was to test the deprecated method that is being deleted in this PR.  The canCheckSingleEvent method was the equivalent of this method for the deprecated checkLogList method.

[the remark this answers is not in the document]
Comment (drivenflywheel) [reply src/test/java/emissary/test/core/junit5/LogbackTesterTest.java]: @nixon124 - you're absolutely right.  Good catch.

**Review by drivenflywheel — APPROVED** [review]

**Pull request merged**